docs/simple_pipelines.md

```diff
@@ -38,6 +38,12 @@ simple-sample-pipeline:
 The remainder of this page shows examples for running from the Docker image. If you
 built from source, you will need to make some modifications to the example commands.
 
+For Data Prepper 2.0 or above, use this command:
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml -v ${PWD} /data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml opensearchproject/data-prepper:latest
```

Comment:
I'd really like to see us start mapping directories for `pipelines/` and `config/`. This will let users specify multiple pipeline files in a directory. For the config directory, they can also start to configure other parameters such as certificate/key files without having to add lots of different `-v` parameters.

I'm happy to merge this in for now since it is more correct than what is currently in `main`. But, this would be a good addition to follow-on with.